.github/workflows/cd.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will build docker images and publish to docker hub
+# For more information see: https://github.com/marketplace/actions/build-and-push-docker-images
+
+name: CD
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set output
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
```

Comment:
Don't you need to logout when deployment was done?

Reply:
Check: https://github.com/commercetools/commercetools-adyen-integration/runs/1696318350?check_suite_focus=true 
Check the step:
<img width="1158" alt="Screenshot 2021-01-20 at 13 50 12" src="https://user-images.githubusercontent.com/3469524/105177084-75e8ad80-5b26-11eb-9eeb-a13ca5ab7599.png">

Reply:
nice^^